### ISSUE TITLE
[8.0.2] fileglobalgc.php errors

### ISSUE THREAD

**Issue by cdamken** [owncloud/core#16468]
I have in the log file many errors like this (3,600,000, about 97 % of my log file in 2 months): 
```
{"reqId":"b9f5de9dfc013b85eabebd3a27f5e84e","remoteAddr":"134.76.10.62","app":"PHP","message":"unlink() expects parameter 2 to be resource, integer given at \/var\/www\/owncloud\/lib\/private\/cache\/fileglobalgc.php#50","level":3,"time":"2015-05-19T11:50:05+00:00"}
{"reqId":"b9f5de9dfc013b85eabebd3a27f5e84e","remoteAddr":"134.76.10.62","app":"PHP","message":"unlink() expects parameter 2 to be resource, integer given at \/var\/www\/owncloud\/lib\/private\/cache\/fileglobalgc.php#50","level":3,"time":"2015-05-19T11:50:06+00:00"}
{"reqId":"b9f5de9dfc013b85eabebd3a27f5e84e","remoteAddr":"134.76.10.62","app":"PHP","message":"unlink() expects parameter 2 to be resource, integer given at \/var\/www\/owncloud\/lib\/private\/cache\/fileglobalgc.php#50","level":3,"time":"2015-05-19T11:50:06+00:00"}
```

https://github.com/owncloud/core/blob/v8.0.2/lib/private/cache/fileglobalgc.php#L50

Could you give me some hints how to solve it?

@MorrisJobke 

00003190

**Comment by MorrisJobke** [comment]
Fixed in 8.0.3 and 8.1 #15079

**Comment by cdamken** [comment]
Super! Thanks!
